modules/core/src/test/scala/gem/LocationSpec.scala

```diff
@@ -70,7 +70,7 @@ class LocationSpec extends FlatSpec with Matchers with PropertyChecks with Arbit
     }
   }
 
-  it should "produce a sorted list of Location.Middle" in {
+  ignore should "produce a sorted list of Location.Middle" in {
```

Comment:
The tail call rewrite is independent of the runtime … it happens way before you know whether you're targeting JVM or JS. So this is suspicious.